src/buttons/buttons.js

```diff
@@ -24,6 +24,10 @@ angular.module('ui.bootstrap.buttons', [])
 
       //ui->model
       element.bind(buttonsCtrl.toggleEvent, function () {
+        if ('disabled' in attrs) {
+          return;
```

Comment:
The issue is Angular 1.4 - it turns out that `attrs.disabled` is `undefined` there.